appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
```

Comment:
I'm not sure what I'm meant to be checking out, are you politely telling me we should just use secure variables and be done with it even though they are sort of ugly? If so, I'll give in otherwise it seems I'm fighting against the grain of how AppVeyor wants to work even though secrets don't belong in source control.

How do we test this before going live against nuget.org, since you can't delete packages just unlisted them? MyGet?